flow360/component/simulation/translator/solver_translator.py

```diff
@@ -503,10 +504,19 @@ def actuator_disk_translator(model: ActuatorDisk):
     }
 
 
-def heat_transfer_entity_info_serializer(volume, solid_zone_boundaries: set):
-    """Heat transfer entity info serializer"""
-    for boundary in volume.private_attribute_zone_boundary_names.items:
-        solid_zone_boundaries.add(boundary)
+def get_solid_zone_boundaries(volume, solid_zone_boundaries: set):
+    """Store solid zone boundaries in a set"""
+    if volume.private_attribute_zone_boundary_names is not None:
+        for boundary in volume.private_attribute_zone_boundary_names.items:
+            solid_zone_boundaries.add(boundary)
+    else:
+        raise Flow360TranslationError(
+            f"boundary_name is required but not found in"
+            f"`{volume.__name__}` instances. \n[For developers]: This error message should not appear."
+            "SimulationParams should have caught this!!!",
+            input_value=volume,
```

Comment:
please provide location in Flow360TranslationError as this is the main reason why we have this class.